src/hooks/useLinkedNote.ts

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useState } from "react";
+import { useDeskproAppClient, useDeskproLatestAppContext } from "@deskpro/app-sdk";
+import { createNote } from "@/services/wrike";
+import type { TaskType } from "@/types";
+import type { ITaskFromList, INote } from "@/services/wrike/types";
+
+export type Result = {
+  isLoading: boolean,
+  addLinkNote: (task: ITaskFromList|TaskType) => Promise<INote|null>,
+  addUnlinkNote: (task: ITaskFromList|TaskType) => Promise<INote|null>,
+};
+
+const getHTMLLink = (link: string) => {
+  return `<a href="${link}">${link}</a>`;
```

Comment:
Why HTML as text instead of ReactNode

Reply:
Notes in Wrike are stored as HTML strings.